releasenotes/notes/qiskit-1.0-incompatibility-9ad22472837837d8.yaml

```diff
@@ -1,19 +1,25 @@
 ---
 prelude: |
     Qiskit 0.45.3 is a point release with no code changes other than to raise an :exc:`ImportError`
-    if it detects it has been installed in an invalid environment with Qiskit 1.0+.
+    if it detects it has been installed in an invalid environment with Qiskit >=1.0.
 
     Please read `our migration guide about the new packaging <https://qisk.it/1-0-packaging-migration>`__
     for help on errors, preparing for Qiskit 1.0, and more detailed background information.
 
+    .. note::
+
+      Qiskit 1.0 is due to release approximately two weeks after Qiskit 0.45.3,
```

Comment:
```suggestion
      Qiskit 1.0 is scheduled to release approximately two weeks after Qiskit 0.45.3,
```

Reply:
If you don't feel too strongly, maybe let's just leave it - there's tons of stuff waiting for Azure at the moment, and even if we jump the queue, it's still a bit full.